hadoop-ozone/dist/src/main/compose/ozone-csi/docker-config

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CORE-SITE.XML_fs.defaultFS=ofs://om
```

Comment:
Why is this line required to use META-INF based discovery?

Reply:
BTW why do we need defaultFS for the CSI test? How is it used.

Reply:
Hi @elek , in most automated robot tests `fs.defaultFS` isn't really used.
Mostly it improves QoL when user logs into the docker-compose cluster and manually checks for something (e.g. `docker-compose exec om /bin/bash` then `ozone fs -ls /`).

Reply:
> Mostly it improves QoL when user logs into the docker-compose cluster and manually checks for something (e.g. docker-compose exec om /bin/bash then ozone fs -ls /).

Good point. And I agree to support debug `ofs` is better choice as all the volumes are visible by default.

Reply:
Thanks @elek . I will file another jira to add those trivial docker-config changes then.

Reply:
Filed https://issues.apache.org/jira/browse/HDDS-4161